### PR TITLE
Fix illegal invocation of `console.log` in example code

### PR DESCRIPTION
**Pull request by josdejong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,7 +137,9 @@ if (socketjs.isSupported()) {
   var socket = socketjs.connect();
 
   // log messages as they arrive
-  socket.receive('greeting', console.log);
+  socket.receive('greeting', function (data) {
```
Comment (stepchowfun) [inline README.md]: For consistency with the rest of the codebase, can you remove the space after the `function` keyword?

Comment (josdejong) [inline README.md]: sure

**Comment by stepchowfun** [comment]
Thanks @josdejong, I didn't know this was illegal. Are built-in functions not first-class values?

**Comment by josdejong** [comment]
I've removed the space

**Comment by josdejong** [comment]
I guess `console.log` need the right context, so you would have to bind to `console`, like `console.log.bind(console)`.

**Comment by stepchowfun** [comment]
Oh, I see.

**Comment by nstepien** [comment]
FWIW you don't need to bind `console.log` in Node, just in browsers.

**Pull request merged**